packages/client/src/__tests__/integration/happy/not-so-exhaustive-schema-mongo/__snapshots__/library.test.ts.snap

```diff
@@ -2284,35 +2284,35 @@ export namespace Prisma {
     * Execute a query given as SQL, interpolating the given parameters,
     * and returning the type-aware result set of the query.
     */
-    queryRaw(params: Query): $Utils.JsPromise<Result<ResultSet>>
+    queryRaw(params: Query): $Utils.JsPromise<NapiResult<ResultSet>>
```

Comment:
I think we can just use fully-qualified name (with namespace here): `Prisma.Result`